packages/safe-core-sdk-utils/src/eip-3770/config.ts

```diff
@@ -0,0 +1,38 @@
+interface NetworkShortName {
+  shortName: string
+  chainId: number
+}
+
+export const networks: NetworkShortName[] = [
```

Comment:
Are these related to contracts we currently have deployed?

Reply:
Yes, I took them from `safe-deployments`